cifar10-multiannotator/utils/model_training.py

```diff
@@ -0,0 +1,272 @@
+import sys
+
```

Comment:
In the notebook you start numbering the consensus labels from -1 which may be a little confusing?  What do you think if we number the rounds like this in both "consensus accuracy" and "model accuracy" plots?

Round k = consensus labels after k rounds of this iterative process of establishing consensus labels and then fitting model to these labels.  
We could consider at k= 0 rounds (base-case) to be the setting where there is no model yet, so consensus is established via majority vote, and then model accuracy is subsequently established by training the model on these consensus labels.  This way both "model accuracy" and "consensus accuracy" plots can have x-axis start at 0 and be lined up?

Reply:
Also if we hardcode a limit on the number of rounds (say 3-10), then I'm not sure if we need the bottom plot showing how many labels changed in each round?  What are your thoughts about including/excluding that and generally imposing a tighter maximum number of rounds?

Reply:
Removing bottom plots, that was just metadata for finding good # of rounds I also wanted to share with you!